lib/src/default_extension_map.dart

```diff
@@ -10,7 +10,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   '3gp': 'video/3gpp',
   '7z': 'application/x-7z-compressed',
   'aab': 'application/x-authorware-bin',
-  'aac': 'audio/x-aac',
+  'aac': 'audio/aac',
```

Comment:
reference for this change?

Reply:
https://www.iana.org/assignments/media-types/audio/aac